### ISSUE TITLE
fzf-maps-n keymap in Lua config

### ISSUE THREAD

**Issue by jrock2004** [junegunn/fzf.vim#1268]
- [x] I have fzf 0.23.0 or above
- [x] I have read through https://github.com/junegunn/fzf.vim/blob/master/README.md
- [x] I have read through https://github.com/junegunn/fzf/blob/master/README-VIM.md
- [x] I have read through the manual page of fzf (`man fzf`)
- [x] I have searched through the existing issues

I am trying to convert my neovim config to lua and trying to figure how I could convert the following to work with lua and paq instead of vim and plug

```
nmap <leader><tab> <plug>(fzf-maps-n)
```

I tried

```
map('n', '<leader><tab>', '<plug>(fzf-maps-n)', {})
```

But I think plug was specific to vim-plug

**Comment by bruhtus** [comment]
> I think plug was specific to vim-plug

No, `<Plug>` is not specific to vim-plug. You can check it with `:help <Plug>`

Maybe you can do something like this:
```vim
map('n', '<leader><tab>', '<plug>(fzf-maps-n)', {noremap = false})
```
because you can't use `<Plug>` mapping with `noremap` option (I'm not sure the default noremap value with lua mapping in neovim). And also I think you should ask those question on neovim repo rather than this repo, you would get an answer faster that way.
